tap_pardot/client.py

```diff
@@ -16,6 +20,8 @@ def __init__(self, message, response_content):
 
 
 def is_not_retryable_pardot_exception(exc):
+    if isinstance(exc, Pardot5XXError):
+        return True
```

Comment:
This should be `False` due to backoff's strange negation in its parameters. Meaning "is not not a retryable pardot exception" 🙄